src/assisted_test_infra/test_infra/controllers/iscsi_target.py

```diff
@@ -0,0 +1,160 @@
+
+from assisted_test_infra.test_infra import utils
+
+from service_client import log
+
+
+class Iqn:
+
+    def __init__(self, unique_name: str, year_month: str = '2024-11', base_domain: str = 'iscsi.redhat.com'):
+        self.year_month = year_month
+        # base domain set in reverse order.
+        self.base_domain = '.'.join(base_domain.split('.')[::-1])
```

Comment:
```suggestion
        self.base_domain = '.'.join(reversed(base_domain.split('.')))
```
is a bit more readable